js/config/loaders.js

```diff
@@ -16,6 +16,9 @@ module.exports = [
       'css-loader'
     ]
   }, {
+    test: /geojs\/.*\.styl$/,
+    loader: 'style-loader!css-loader!stylus-loader'
```

Comment:
You need to add the loaders to `package.json`.  I believe the full list is:
* `style-loader`
* `stylus-loader`
* `stylus`
* `nib`